control-plane/typeSpecConverter.cpp

```diff
@@ -111,6 +115,49 @@ bool TypeSpecConverter::preorder(const IR::Type_Name* type) {
     return false;
 }
 
+bool TypeSpecConverter::preorder(const IR::Type_Newtype* type) {
+    if (p4RtTypeInfo) {
+        bool orig = true;
+        cstring uri;
+        uint32_t sdnB;
+        auto ann = type->getAnnotation(IR::Annotation::pitranslateAnnotation);
```

Comment:
That's a poor name, why does it reference "pi"?

Reply:
I meant to say "p4runtime_translationAnnotation" - isn't that too long a name?  Anyway, let me me change it.